src/features/dashboard/ui/IndividualResponseViewer.tsx

```
@@ -0,0 +1,82 @@
+import Checkbox from "../../../../design-system/ui/Checkbox";
+import { Order } from "../../ticket/model/ticketInformation";
+import IconButton from '../../../../design-system/ui/buttons/IconButton';
+import rightButton from '../../../../public/assets/main/RightButton.svg';
+import leftButton from '../../../../public/assets/main/LeftButton.svg';
+import UnderlineTextField from '../../../../design-system/ui/textFields/UnderlineTextField';
```

ESLint

[error] 'UnderlineTextField' is defined but never used.

(@typescript-eslint/no-unused-vars)